### DISCUSSION TITLE
Android Auto Support

### DISCUSSION THREAD

**Discussion by JutPie** [PaulWoitaschek/Voice#1265] [Ideas]
I realy love this App and its my daily driver to listen to Audiobooks for a few years  now. In my Van i have a new Radio which is capable of Android Auto. Ist there any Plan to Support Android Auto. I have no Idea if this is a big thing to develop  / to implement.

**Comment by PaulWoitaschek** [comment]
Yes, Android auto is supported. Which version are you using currently?

**Reply by JutPie** [reply]
Hey and thanks for the very quick reply. Im using the newest Beta i think. Its Version 6.0.0 beta02. But after your reply i try it another time and uninstalled and then reinstall the App. And now it worked. So my fault.  By the way amazing App. Never found a better solution for Audiobooks on Android.
